devices.js

```diff
@@ -4227,6 +4251,41 @@ const devices = [
         description: 'GMY Smart bulb, 470lm, vintage dimmable, 2700-6500k, E27',
         extend: generic.light_onoff_brightness_colortemp,
     },
+
+    // Meazon Plug and DinRail meters
+    {
+        zigbeeModel: ['101.301.001649', '102.106.000235', '101.301.001838', '101.301.001802', '101.301.001738',
+            '102.106.001111', '102.106.000348', '101.301.001412', '101.301.001765', '101.301.001814',
+            '102.106.000256', '102.106.001242'],
+        model: 'Plug/DinRail',
```

Comment:
I see that there are various models available, this definition also seems to combine a dinrail meter and plug, could you split them and add the corresponding modelID's?

Reply:
@Koenkk All the Dinrails and the Plugs are the same model; actually, the "modelID" is the serial number of each device :(, but this is what I have got on the debug.
Even if Dinrails and plugs are not the same device, from a zigbee point of view they behave exactly in the same way, so does it make sense to split the config? on the fromZigBee.js you will have noticed the processing is the same, as clusters, ep, ... are the same (I think they share the same firmware). Advise please. Txs.

Reply:
In that case, I think at least the plug and dinrail should be splitted (the plug is a power socket right?).

Reply:
done. Yes, plug is a power socket.